README.md

```diff
@@ -20,7 +20,16 @@ downloads](https://cranlogs.r-pkg.org/badges/pak)](https://www.r-pkg.org/pkg/pak
 Status](https://img.shields.io/codecov/c/github/r-lib/pak/master.svg)](https://codecov.io/github/r-lib/pak?branch=master)
 <!-- badges: end -->
 
-pak installs R packages from various sources.
+pak installs R packages from various sources. It is a replacement for
+`install.packages()` and `devtools::install_github()`. pak is fast, safe
+and
+convenient.
+
+<p align="center">
+
+<img width="1000" src="https://cdn.jsdelivr.net/gh/r-lib/pak@readme/tools/images/install-2.svg">
```

Comment:
remind me how you make these magical svgs again...

Also GH does actually display them with the animations right?

Reply:
asciinema + https://github.com/marionebl/svg-term-cli

Yes, take a look: https://github.com/r-lib/pak/tree/readme#readme

I like `svg-term --window` to include the window details.